assets/sass/modules/_forms.scss

```diff
@@ -189,6 +189,40 @@
       right: 10px;
       opacity: 0.5;
     }
+
+    .password-toggle {
+
+      .button-off {
+        display: none;
+      }
+
+      .button {
+        right: 5px;
+        position: absolute;
+        top: 50%;
+        margin-top: -13px;
+        line-height: 24px;
+        height: 26px;
+        padding: 0 5px;
+        font-size: 11px;
+      }
+
+      .button-on {
+        background: #929292;
```

Comment:
I have not found any suitable color in the _variables.scss for the "show" button.
Should I add a new variable or better to ask designer to change the mock?